src/components/GameCards/GameCardMasonry.vue

```diff
@@ -2,19 +2,41 @@
   <div
     v-if="gameId && games[gameId]"
     :class="gameCardClass"
->
+  >
     <img
       :src="coverUrl"
       :alt="game.name"
       @click="openDetails"
     >
+
+    <game-completed
+      v-if="gameProgress"
+      size="small"
+      :progress="gameProgress"
+      @click.native="openDetails"
+    />
+
+    <game-progress
```

Comment:
Great idea! However, I think the `game-completed` component should live inside of the `game-progress` component and should only render the checkmark in the component, the parent component can control when to show or not the component, you can pass a prop (e.g. `show-badge`) to game-progress so you control when the ✔️ shows up.

Reply:
That would work for `GameCardMasonry`, but components like `GameCardDefault` have the badge outside the `.game-info` div, 1 level above where `game-progress` sits.

I could try turning this into a pseudo element and add a trigger-class for the parent component.

Reply:
Alright, I turned it into a pseudo element. 👍